jetty9-compat-base/src/main/java/com/google/apphosting/vmruntime/jetty9/VmRuntimeWebAppContext.java

```diff
@@ -195,6 +187,28 @@ protected void doStart() throws Exception {
     }
 
     super.doStart();
+
+    // Look for a datastore service within the context
+    ClassLoader orig = Thread.currentThread().getContextClassLoader();
```

Comment:
Is adding filters to the app by the container (without user settings) is something unusual or a standard practice? If the latter then I like it better otherwise feel free to leave as is and a comment explaining why we are doing it, such as the one you provided here, would be great!